src/dom.js

```diff
@@ -777,7 +777,7 @@ export function isAmpElement(element) {
 /**
  * Return a promise that resolve when an AMP element upgrade from HTMLElement
  * to CustomElement
- * @param {!Element} element
+ * @param {?} element
```

Comment:
Why does this change?

Reply:
I switched to HTMLElement. Turns out `Element` doesn't have all the CE callbacks, whereas HTMLElement does.